frontend/src/pages/accounts/profile.page.tsx

```diff
@@ -65,12 +66,20 @@ const Profile: NextPage = () => {
     category: "Purchase Button",
     label: "profile-bottom-promo",
   });
+  const hash = getCookie("profile-location-hash");
+  if (hash) {
+    document.location.hash = hash;
+    clearCookie("profile-location-hash");
+  }
```

Comment:
Not a major issue since it's fairly lightweight code, but as this is a side effect, you probably only want to run it once, rather than every render:

```suggestion
  useEffect(() => {
    const hash = getCookie("profile-location-hash");
    if (hash) {
      document.location.hash = hash;
      clearCookie("profile-location-hash");
    }
  }, []);
```